GoogleMediaFrameworkDemo/GoogleMediaFrameworkTests/GoogleMediaFrameworkDemoTests.m

```diff
@@ -208,7 +217,7 @@ - (void)assertPlaybackDoesProgress {
 
 - (void)assertPlaybackDoesNotProgress {
   NSTimeInterval startMediaTime = [_player currentMediaTime];
-  [self waitForTimeInterval:1];
+  [self waitForTimeInterval:10];
```

Comment:
is this intentional?

Reply:
I don't know why GitHub is insisting that I made this change - in my local client it's still 1, not 10/.